src/service/.eslintrc.js

```diff
@@ -0,0 +1,6 @@
+// eslint-disable-next-line import/no-commonjs
+module.exports = {
```

Comment:
You can do this with a files override in the core `.eslintrc`

Reply:
What do you think about setting it directly on the file instead? Might protect it from renames.
```js
/* eslint local/service-getters: 2 */
```

Reply:
Sure